From: Jan Engelhardt <jengelh@inai.de>

There are some manual commands which create dependency files, and
they do fail if they concurrently try to create the directory
(because it does something like test -e auto || mkdir auto, rather
than checking just the return value of mkdir for EISDIR.)

Anyway, create the directory beforehand.
---
 Makefile.am  |    1 +
 configure.ac |    1 +
 2 files changed, 2 insertions(+)

Index: schismtracker-20240609/Makefile.am
===================================================================
--- schismtracker-20240609.orig/Makefile.am
+++ schismtracker-20240609/Makefile.am
@@ -149,6 +149,7 @@ CLEANFILES = \
 	auto/helptext.c
 
 auto/default-font.c: Makefile.am scripts/bin2h.sh scripts/build-font.sh $(fonts)
+	${MKDIR_P} auto
 	sh $(srcdir)/scripts/build-font.sh $(srcdir) $(fonts) >$@
 
 auto/helptext.c: Makefile.am scripts/genhelp.pl $(helptexts)
Index: schismtracker-20240609/configure.ac
===================================================================
--- schismtracker-20240609.orig/configure.ac
+++ schismtracker-20240609/configure.ac
@@ -111,6 +111,7 @@ AM_PROG_CC_C_O
 AC_PROG_CPP
 AC_PROG_INSTALL
 AC_PROG_LN_S
+AC_PROG_MKDIR_P
 
 dnl do we have Git
 AC_CHECK_TOOL([GIT], [git])
